common/supplemental/units.xml

```diff
@@ -0,0 +1,174 @@
+		<convertUnit source='furlong' target='meter' factor='ft2m*660'/>
+		<convertUnit source='inch' target='meter' factor='ft2m/12'/>
+		<convertUnit source='mile' target='meter' factor='ft2m*5280'/>
+		<convertUnit source='nautical-mile' target='meter' factor='1852'/>
+		<convertUnit source='yard' target='meter' factor='ft2m*3'/>
+		<convertUnit source='ounce' target='kilogram' factor='lb2kg/16'/>
+		<convertUnit source='ounce-troy' target='kilogram' factor='0.03110348'/>
+		<convertUnit source='pound' target='kilogram' factor='lb2kg'/>
+		<convertUnit source='stone' target='kilogram' factor='lb2kg*14'/>
+		<convertUnit source='ton' target='kilogram' factor='lb2kg*2000'/>
+		<convertUnit source='horsepower' target='kilogram-square-meter-per-cubic-second' factor='ft2m * lb2kg * gravity * 550'/>
+		<convertUnit source='inch-hg' target='kilogram-per-meter-square-second' factor='3386.389'/>
+		<convertUnit source='knot' target='meter-per-second' factor='1852/3600'/>
+		<convertUnit source='fahrenheit' target='kelvin' factor='5/9' offset='2298.35/9'/>
+		<convertUnit source='acre-foot' target='cubic-meter' factor='ft2m^3 * 43560'/>
+		<convertUnit source='barrel' target='cubic-meter' factor='cup2m3 * 672'/>
+		<convertUnit source='bushel' target='cubic-meter' factor='0.03523907'/>
+		<convertUnit source='cup' target='cubic-meter' factor='cup2m3'/>
+		<convertUnit source='fluid-ounce' target='cubic-meter' factor='cup2m3/8'/>
+		<convertUnit source='gallon' target='cubic-meter' factor='cup2m3*16'/>
+		<convertUnit source='tablespoon' target='cubic-meter' factor='cup2m3/16'/>
+		<convertUnit source='teaspoon' target='cubic-meter' factor='cup2m3/48'/>
+		<convertUnit source='karat' target='portion' factor='1/24'/>
+		<convertUnit source='pint' target='cubic-meter' factor='cup2m3 * 2'/>
+		<convertUnit source='quart' target='cubic-meter' factor='cup2m3 * 4'/>
+		<convertUnit source='fluid-ounce-imperial' target='cubic-meter' factor='0.00002841306'/>
+		<convertUnit source='gallon-imperial' target='cubic-meter' factor='0.00454609'/>
+		<convertUnit source='dunam' target='square-meter' factor='1000'/>
+		<convertUnit source='mile-scandinavian' target='meter' factor='10000'/>
+		<convertUnit source='hectare' target='square-meter' factor='10000'/>
+		<convertUnit source='joule' target='kilogram-square-meter-per-square-second' factor='1'/>
+		<convertUnit source='newton' target='kilogram-meter-per-square-second' factor='1'/>
+		<convertUnit source='carat' target='kilogram' factor='0.0002'/>
+		<convertUnit source='gram' target='kilogram' factor='0.001'/>
+		<convertUnit source='metric-ton' target='kilogram' factor='1000'/>
+		<convertUnit source='watt' target='kilogram-square-meter-per-cubic-second' factor='1'/>
+		<convertUnit source='bar' target='kilogram-per-meter-square-second' factor='100000'/>
+		<convertUnit source='pascal' target='kilogram-per-meter-square-second' factor='1'/>
+		<convertUnit source='celsius' target='kelvin' factor='1' offset='-273.15'/>
```

Comment:
I believe the sign of the offset for `celsius` should by `+`. ie `273.15` to be consistent with the direction of the other offset in `fahrenheit` (that being the only other offset so far.  In any case, both signs should agree.